Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1,74 +1,245 @@
+    | Incident type | Incident type.                                                                                                                                                              | False |
 
-- The steps should be performed in order to ensure no communication between the services is lost.
-- Refer to the [Setup Examples](#setup-examples) for information regarding advanced setups.
+### Authentication Using the Client Credentials Flow
+
+Note ⚠️: The **chat commands** are only supported when using the `Authorization Code flow`.
+
+1. Choose the 'Client Credentials' option in the *Authentication Type* parameter.
+2. Enter your Client/Application ID in the *Bot ID* parameter.
+3. Enter your Client Secret in the *Bot Password* parameter.
+4. Set the *Default team* and the *Notifications channel* parameters.
+5. Set the *Long running instance* parameter to 'True'.
+6. Save the instance.
+7. Click **Test** to validate the URLs, token, and connection.
+8. Configure the messaging endpoint if you haven't done so already (Step #9 in the [Bot Creation](#1-create-the-demisto-bot-in-microsoft-teams) section).
+9. [Add the Demisto Bot to a Team](#4-add-the-demisto-bot-to-a-team).
+
+### Authentication Using the Authorization Code Flow
+
+Note ⚠️: The [microsoft-teams-ring-user](https://learn.microsoft.com/en-us/graph/api/application-post-calls?view=graph-rest-1.0&tabs=http) command requires authenticating with `Client Credentials`.
+
+1. Choose the 'Authorization Code' option in the *Authentication Type* parameter.
+2. Enter your Client/Application ID in the *Bot ID* parameter.
+3. Enter your Client Secret in the *Bot Password* parameter.
+4. Enter your Application redirect URI in the *Application redirect URI* parameter (Step #7 in the [Grant Permissions](#2-grant-the-demisto-bot-permissions-in-microsoft-graph) section).
+5. Set the *Default team* and the *Notifications channel* parameters.
+6. Set the *Long running instance* parameter to 'True'.
+7. Save the instance.
+8. Configure the messaging endpoint if you haven't done so already (Step #9 in the [Bot Creation](#creating-the-demisto-bot-using-microsoft-azure-portal) section).
+9. [Add the Demisto Bot to a Team](#add-the-demisto-bot-to-a-team)
+10. Run the ***!microsoft-teams-generate-login-url*** command in the Playground and follow the on-screen instructions.
+11. Save the instance.
+12. Run the ***!microsoft-teams-auth-test*** command. A 'Success' message should be printed to the War Room.
+
+If you encounter any issue or error — especially a message saying **"Tenant ID is not found"** — please refer to the [Troubleshooting](#Troubleshooting) section.
+
+### 4. Add the Demisto Bot to a Team
+
+- The following needs to be done after configuring the integration on Cortex XSOAR/XSIAM (the previous step).
```

Comment:
After configuring the integration on Cortex XSOAR/XSIAM (the previous step), do the following: